source/Xunit.Gherkin.Quick.Tests/ParameterHelperTests.cs

```diff
@@ -0,0 +1,103 @@
+using FluentAssertions;
+using Gherkin.Ast;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace Xunit.Gherkin.Quick.Tests
+{
+    public class ParameterHelperTests
+    {
+        [Fact]
+        public void GetParamValues_NoTableData()
+        {
+            var method = typeof(SimpleTestClass).GetMethod("NoTableData1");
+
+            var step = new Step(null, "Given", "some data", null);
+
+            var specParams = new List<string>
+            {
+                "String Value 1",
+                "250"
+            };
+
+            var values = ParameterHelper.GetParamValues(method, step, specParams);
+
+            values.Should().NotBeNull()
+                .And.HaveCount(2);
+            values[0].Should().Be("String Value 1");
+            values[1].Should().Be(250);
+        }
+
+        [Fact]
+        public void GetParamValues_TableDataAtStart()
+        {
+            var method = typeof(SimpleTestClass).GetMethod("TableDataAtStart1");
+
+            var step = new Step(null, "Given", "some data", new DataTable(new[] { new TableRow(null, new[] { new TableCell(null, "A value") }) }));
+
+            var specParams = new List<string>
+            {
+                "String Value 2",
+                "251"
+            };
+
+            var values = ParameterHelper.GetParamValues(method, step, specParams);
+
+            values.Should().NotBeNull()
+                .And.HaveCount(3);
+            values[0].Should().BeOfType<DataTable>();
```

Comment:
The unit tests are fine. Just also assert that this datatable is the same as the one you passed in. That should be it.